types/logger/logger.go

```diff
@@ -284,3 +284,47 @@ type rateLimitContext struct {
 	noopFormatter
 	context string
 }
+
+// ApplyPostProcess works with PostProcess to allow
```

Comment:
I read this godoc several times but still don't get what it does from its description.

Why is this exported if it seems like just an implementation detail of PostProcess which has understandable docs?

Reply:
I rewrote. Please let me know if it remains unclear.